spk/minio/Makefile

```diff
@@ -1,5 +1,5 @@
 SPK_NAME = minio
-SPK_VERS = 2019-08-21T19-40-07Z
+SPK_VERS = 2019-09-26T19-42-35Z
 SPK_REV = 1
```

Comment:
please regard:
SPK_REV = Package Revision number. 
Starts at 1, goes up 1 for every subsequent release

Reply:
Thanks, I've missed that. Fixed in https://github.com/SynoCommunity/spksrc/pull/3779/commits/a00d251904d7995a4aedb0ea35102ab2ba5dab18